brave/brave-ios
# Fix #8329: Send to your Devices action "Send" button placement

## Summary of Changes


This pull request fixes #8329 

The Send Button is removed from button and added to navigation bar as action button. Also Indicator are flashed in order to warn user if the content doesnt fit desired frame size - height.

## Submitter Checklist:


## Test Plan:


1. Open Brave
2. Enable Sync / join a Sync Chain
3. Open a website
4. Press action for Send To Your Device
5. Check Send button is right side of navigation bar

Send necessary tab

## Screenshots:


![12111](https://github.com/brave/brave-ios/assets/6643505/22bdea68-e27e-4b53-82af-330cca473047)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
